test/MUXControlsTestApp/MSTest/MUXControlsInnerLoopTestApp.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <UnitTestPlatformVersion Condition="'$(UnitTestPlatformVersion)' == ''">$(VisualStudioVersion)</UnitTestPlatformVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
+    <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>
+    <AppxBundle>Never</AppxBundle>
+    <AppInstallerUpdateFrequency>1</AppInstallerUpdateFrequency>
+    <AppInstallerCheckForUpdateFrequency>OnApplicationRun</AppInstallerCheckForUpdateFrequency>
+  </PropertyGroup>
+  <!-- We need to define NuGet references using this newer style for running tests locally in VS to work.
+       However, for some reason, specifially in lab builds, using the below to resolve NuGet references
+       causes it to complain that the uap10.0.15063 framework wasn't referenced.
+       As such, we'll use the more old-school package.json NuGet reference format for lab builds. -->
+  <PropertyGroup Condition="'$(XES_DFSDROP)' == ''">
+    <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(XES_DFSDROP)' == ''">
+    <PackageReference Include="MUXCustomBuildTasks">
+      <Version>1.0.48</Version>
+    </PackageReference>
```

Comment:
Is the manifest the only difference ? if so can we do a conditional on the csproj file to choose the inner loop manifest if the current solution is inner loop solution ? That way we avoid duplicating the csproj files ?

Reply:
Yes I think the only difference we need is in the manifest file. However I am not quite sure how you would use conditional compiling. Is there a way to detect in which solution we are from the CS-PROJ?

Reply:
Yes. In the project file, for any of the tags, you can add conditions like so..
 <AppxManifest Include="PackageInnerLoop.appxmanifest"Condition="$(SolutionName) == 'MUXControlsInnerLoop'">

You can do the same to only update the properties/items that need to be different for the inner loop solution.

Reply:
Didn't thought about that. I have updated the files accordingly with latest commit. Thanks!

Reply:
Cool. Looks great. did a minor change to the condition so that we don't end up in cases where no manifest gets included.